urns/urns_test.go

```diff
@@ -292,10 +292,12 @@ func TestTelURNs(t *testing.T) {
 		{"62877747666", "ID", "tel:+62877747666", false},
 		{"0877747666", "ID", "tel:+62877747666", false},
 		{"07531669965", "GB", "tel:+447531669965", false},
-		{"12345", "RW", "", true},
-		{"0788383383", "", "", true},
-		{"0788383383", "ZZ", "", true},
-		{"MTN", "RW", "", true},
+		{"12345", "RW", "tel:12345", false},
+		{"0788383383", "", "tel:0788383383", false},
+		{"0788383383", "ZZ", "tel:0788383383", false},
+		{"PRIZES", "RW", "tel:prizes", false},
```

Comment:
add some cases with spaces here maybe?

Looking at what we do in Python, it seems the big missing difference is that in `normalizeNumber` if we aren't able to validate the number as a REAL number via phonenumbers, then we normalize the number by removing non (a-z0-9) chars and return that. That's even weaker than what we are doing here, but I guess the idea is that maybe the idea of whether a number is a "valid" phone number is beyond the purview of this library. The end result of that is basically NewTelURL() always works but maybe that's ok? Its job is just to normalize?

Reply:
I'm looking at `normalizeNumber` here and `normalize_number` in RP and I think they're equivalent - the former returns something that already has had non (a-z0-9) chars stripped out.

Have added a test with spaces